src/status_im/events.cljs

```diff
@@ -2149,3 +2152,119 @@
  :shake-event
  (fn [cofx _]
    (logging/show-logs-dialog cofx)))
+
+(re-frame/reg-fx
```

Comment:
this namespace is only for `register-handler-fx`, and they should only contain simple events that call a function.
this would belong to ens.core

Reply:
:+1: